src/tr.i18n.js

```diff
@@ -0,0 +1,541 @@
+module.exports = {
+  'menu.file': {
+    'New Window': 'Yeni Pencere',
+    'File': 'Dosya',
+    'New': 'Yeni',
+    'Open…': 'Aç…',
+    'Save': 'Kaydet',
+    'Save As…': 'Farklı Kaydet…',
+    'Clipboard': 'Pano',
+    'Load SGF': 'SGF Yükle',
+    'Copy SGF': 'SGF Kopyala',
+    'Copy ASCII Diagram': 'ASCII Diyagramı Kopyala',
+    'Game Info': 'Oyun Bilgisi',
+    'Manage Games…': 'Oyunları Yönet…',
+    'Preferences…': 'Tercihler…'
+  },
+  'updater': {
+    'Download Update': 'Güncellemeyi İndir',
+    'View Changelog': 'Değişiklikleri Görüntüle',
+    'Not Now': 'Şimdi Değil',
+    '${appName} v${version} is available now.': '${appName} v${version} şimdi mevcut',
+    'OK': 'Tamam',
+    'No updates available': 'Güncelleme mevcut değil',
+    '${appName} v${version} is the latest version.': '${appName} v${version} en son sürümdür'
+  },
+  'exception': {
+    '${appName} v${version}': '${appName} v${version}',
+    'Something weird happened. ${appName} will shut itself down. If possible, please report this on ${appName}’s repository on GitHub.': 'Garip bir şey oldu. ${appName} kendini kapatacak. Eğer mümkünse, lütfen bunu GitHub’da ${appName} deposunda bildirin.'
+  },
+  'menu.play': {
+    'Play': 'Oyna',
+    'Toggle Player': 'Oyuncu Değiştir',
+    'Select Point': 'Nokta Seçin',
+    'Enter a coordinate to select a point': 'Nokta seçmek için koordinat girin',
+    'Pass': 'Pas',
+    'Resign': 'Terk',
+    'Estimate': 'Tahmin',
+    'Score': 'Skor'
+  },
+  'menu.edit': {
+    'Edit': 'Düzenle',
+    'Undo': 'Geri Al',
+    'Redo': 'İleri Git',
+    'Toggle Edit Mode': 'Düzenleme Moduna Geç',
+    'Select Tool': 'Araç Seç',
+    'Stone Tool': 'Taş Aracı',
+    'Cross Tool': 'Çarpı Aracı',
+    'Triangle Tool': 'Üçgen Aracı',
+    'Square Tool': 'Kare Aracı',
+    'Circle Tool': 'Daire Aracı',
+    'Line Tool': 'Çizgi Aracı',
+    'Arrow Tool': 'Ok Aracı',
+    'Label Tool': 'Yazı Aracı',
+    'Number Tool': 'Sayı Aracı',
+    'Copy Variation': 'Varyasyonu Kopyala',
+    'Cut Variation': 'Varyasyonu Kes',
+    'Paste Variation': 'Varyasyonu Yapıştır',
+    'Make Main Variation': 'Ana varyasyon Yap',
+    'Shift Left': 'Sola Geç',
+    'Shift Right': 'Sağa Geç',
+    'Flatten': 'Düzleştir',
+    'Remove Node': 'Düğümü Kaldır',
+    'Remove Other Variations': 'Diğer Varyasyonları Sil'
+  },
+  'menu.find': {
+    'Find': 'Bul',
+    'Toggle Find Mode': 'Bulma Moduna Geç',
+    'Find Next': 'Sonrakini Bul',
+    'Find Previous': 'Öncekini Bul',
+    'Toggle Hotspot': 'Hotspot ->',
```

Comment:
This translation is questionable, can you please make sure this is correct?

Reply:
hotspot and toggle words don't have a direct translation on Turkish. Now I found a way to tanslate.